packages/tools/fluid-runner/src/test/fluidRunner.spec.ts

```diff
@@ -37,7 +37,8 @@ describe("fluid-runner from command line", () => {
 					`--inputFile=${snapshot}`,
 					`--outputFile=${outputFilePath}`,
 					`--telemetryFile=${telemetryFile}`,
-					`--telemetryFormat=CSV`,
+					"--telemetryFormat=CSV",
+					"--eventsPerFlush=-2",
```

Comment:
If we accept negative values, what's the semantics of that? I see by default it's 50 and if it's negative it has the same meaning as being 0, so maybe we should just take in >=0 ?

Reply:
Yes, negative value works exactly like 0. I didn't want to bloat the code with parsing since it won't really break anything.